Mathlib/Analysis/SpecialFunctions/Complex/Arg.lean

```diff
@@ -227,6 +227,10 @@ theorem tan_arg (x : ℂ) : Real.tan (arg x) = x.im / x.re := by
 theorem arg_ofReal_of_nonneg {x : ℝ} (hx : 0 ≤ x) : arg x = 0 := by simp [arg, hx]
 #align complex.arg_of_real_of_nonneg Complex.arg_ofReal_of_nonneg
 
+@[simp, norm_cast]
+lemma ofNat_arg {n : ℕ} : arg n = 0 :=
```

Comment:
This is `Nat.cast`, not `OfNat.ofNat`. Please also add a lemma about `OfNat.ofNat`, so that Lean can simplify numerical constants.

Reply:
OK.